### DISCUSSION TITLE
Max Row Size and performance

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#16824] [Archive]
I am starting to think about to create a table where each user has a list of important connections: from which building does he opened which other building.

considering that I have around 36.000 users and only 200 building this would end up at:

200*200*36.000 if every user would have a full list. This results in 1.400.000.000 records.

can Directus handle this size of rows? Or should I think about a better solution?

**Comment by NilsBaumgartner1994** [comment]
I think for my problem itself the better way would be to have a sigle collection with all 200*200 connections and then another collection with a user to that previous building to building connection. Allowing me to count the connections easier.

the original question still is important for me, since postgre has „no limit“ (limit at 64TB or so)

**Comment by rijkvanzanten** [comment]
It really depends on database and server performance at that point! It also really matters what you're planning on doing with these records, and how many are in a single table at a given time.
